src/AppChooser/Dialog.vala

```diff
@@ -110,7 +110,9 @@ public class AppChooser.Dialog : Hdy.Window {
         placeholder.show_all ();
 
         listbox = new Gtk.ListBox () {
-            expand = true
+            vexpand = true,
+            hexpand = true,
+            activate_on_single_click = false
```

Comment:
I'm not sure about unsetting `activate_on_single_click`, i would prefer that we handle activating/selecting the row by ourselves in the `::row_activated` signal.